### PR TITLE
[ENGDOCS-1012] Get started refresh part 1 & 2

### PR DESCRIPTION
**Pull request by craig-osterhout**
### Proposed changes

Updates to part 1 and part 2 of getting started.

### Related issues (optional)
ENGDOCS-1012

### CODE REVIEW

File: get-started/02_our_app.md [hunk 1]
```diff
@@ -6,39 +6,37 @@ redirect_from:
 description: Containerize and run a simple application to learn Docker
 ---
 
+For rest of this guide, you will be working with a simple todo
```
Comment (jerae-duffin) [inline get-started/02_our_app.md:9]:
```suggestion
For the rest of this guide, you will be working with a simple todo
```

File: get-started/02_our_app.md [hunk 2]
```diff
@@ -6,39 +6,37 @@ redirect_from:
...
+- A conceptual understanding of [containers and images](../get-started/overview.md/#docker-objects).
 
 ## Get the app
 
-Before we can run the application, we need to get the application source code onto 
-our machine. For real projects, you will typically clone the repo. But, for this tutorial,
-we have created a ZIP file containing the application.
+Before you can run the application, you need to get the application source code onto your machine.
+
+1. Clone the [getting-started repository](https://github.com/docker/getting-started/tree/master){:target="_blank" rel="noopener" class="_"} using the following command.
```
Comment (jerae-duffin) [inline get-started/02_our_app.md:24]:
```suggestion
1. Clone the [getting-started repository](https://github.com/docker/getting-started/tree/master){:target="_blank" rel="noopener" class="_"} using the following command:
```

File: get-started/02_our_app.md [hunk 3]
```diff
@@ -49,26 +47,34 @@ Dockerfile is simply a text-based file with no file extension. A Dockerfile cont
...
+    Create an empty file named `Dockerfile`.
+    ```console
+    $ type nul > Dockerfile
+    ```
     <hr>
    </div>
    </div>
 
-2. Add the following contents to the Dockerfile. If you've created Dockerfiles before, you might see a few flaws in the Dockerfile below. But, don't worry. We'll go over them.
+2. Using a text editor or code editor, add the following contents to the Dockerfile.
```
Comment (jerae-duffin) [inline get-started/02_our_app.md:77]:
```suggestion
2. Using a text editor or code editor, add the following contents to the Dockerfile:
```

File: get-started/02_our_app.md [hunk 4]
```diff
@@ -80,72 +86,63 @@ Dockerfile is simply a text-based file with no file extension. A Dockerfile cont
...
-(this tutorial and your freshly launched app container).
+If you take a quick look at your Docker Dashboard, you should see at least one container running that is using the `getting-started` image and on port `3000`.
 
 ![Docker Dashboard with tutorial and app containers running](images/dashboard-two-containers.png)
 
 ## Next steps
 
-In this short section, you learned the basics about building a container image and created a
-Dockerfile to do so. Once you built an image, you started the container and saw the running app.
+In this short section, you learned the basics about creating a  Dockerfile to build a container image. Once you built an image, you started a container and saw the running app.
```
Comment (jerae-duffin) [inline get-started/02_our_app.md:144]:
```suggestion
In this short section, you learned the basics about creating a Dockerfile to build a container image. Once you built an image, you started a container and saw the running app.
```

File: language/nodejs/build-images.md [hunk 5]
```diff
@@ -10,12 +10,12 @@ redirect_from:
...
 ## Prerequisites
 
-* You understand basic Docker concepts, and you have completed the orientation
-  and setup in Get started [Part 1](../../get-started/index.md).
+* You understand basic [Docker concepts](../../get-started/overview.md).
 * You're familiar with the [Dockerfile format](../../build/building/packaging.md#dockerfile).
 * You have [enabled BuildKit](../../build/buildkit/index.md#getting-started)
   on your machine.
 
+
```
Comment (jerae-duffin) [inline language/nodejs/build-images.md:18]: Extra line?

File: get-started/02_our_app.md (hunk 1 above)
Comment (craig-osterhout) [reply get-started/02_our_app.md]: Thanks. Fixed in the commit below.

File: get-started/02_our_app.md (hunk 2 above)
Comment (craig-osterhout) [reply get-started/02_our_app.md]: Thanks. Fixed in the commit below.

File: get-started/02_our_app.md (hunk 3 above)
Comment (craig-osterhout) [reply get-started/02_our_app.md]: Thanks. Fixed in the commit below.

File: get-started/02_our_app.md [hunk 6]
```diff
@@ -80,72 +86,63 @@ Dockerfile is simply a text-based file with no file extension. A Dockerfile cont
...
    EXPOSE 3000
    ```
-  > **Note**
-  >
-  > Select an instruction in the Dockerfile example to learn more about the instruction.
+   > **Note**
+   >
+   > Select an instruction in the Dockerfile example to learn more about the instruction.
+
+3. Build the container image using the following commands.
```
[the remark this answers is not in the document]
Comment (craig-osterhout) [reply get-started/02_our_app.md]: Thanks. Fixed in the commit below.

File: get-started/02_our_app.md (hunk 4 above)
Comment (craig-osterhout) [reply get-started/02_our_app.md]: Thanks. Fixed in the commit below.

File: language/java/build-images.md [hunk 7]
```diff
@@ -8,12 +8,12 @@ description: Learn how to build your first Docker image by writing a Dockerfile
...
 ## Prerequisites
 
-* You understand basic Docker concepts, and you have completed the orientation
-  and setup in Get started [Part 1](../../get-started/index.md).
+* You understand basic [Docker concepts](../../get-started/overview.md).
 * You're familiar with the [Dockerfile format](../../build/building/packaging.md#dockerfile).
 * You have [enabled BuildKit](../../build/buildkit/index.md#getting-started)
   on your machine.
 
+
```
[the remark this answers is not in the document]
Comment (craig-osterhout) [reply language/java/build-images.md]: Thanks. Fixed in the commit below.

File: language/nodejs/build-images.md (hunk 5 above)
Comment (craig-osterhout) [reply language/nodejs/build-images.md]: Thanks. Fixed in the commit below.

**Review by jerae-duffin — APPROVED** [review]
LGTM!

**Pull request merged**